### DISCUSSION TITLE
buy/sell at specific date

### DISCUSSION THREAD

**Discussion by sgs1** [kernc/backtesting.py#439] [Q&A]
Hi guys!

I've a CSV file with all the required columns; but, I'd like to buy at specific date and sell at specific date.
For example: sell every monday and buy every Thursday.
Is this possible with this python software?
Thanks.

**Comment by AGG2017** [comment]
Many ways to do it. I simply add another column to the dataframe: df['dow'] = df.index.dayofweek
Later in your `next` function check if self.data['dow'][-1]==x to do the processing. Monday=0 ... Sunday=6
This way it is a bit faster when using complex optimizations (dow is calculated just once in advance).
